python/lsst/validate/drp/util.py

```diff
@@ -140,6 +148,32 @@ def sphDist(ra1, dec1, ra2, dec2):
     return dist
 
 
+def averageRaFromCat(cat):
```

Comment:
Though validate_drp isn't a particularly time-critical operation, I do worry that your use of these "only RA" , "only Dec" functions is slow, given you're throwing away the other average. I see you're using it in the aggregator for TEx: how many values are typically input?